### PR TITLE
docs: add patches and integrations in prerequisites

### PR DESCRIPTION
**Pull request by taku-nm**
in response to users missing that step

### CODE REVIEW

File: docs/0_prerequisites.md [hunk 1]
```diff
@@ -7,9 +7,12 @@ To use ReVanced CLI, you will need to fulfill certain requirements.
 - Java SDK 11 (Azul JDK or OpenJDK)
 - [Android Debug Bridge (adb)](https://developer.android.com/studio/command-line/adb) if you want to deploy the patched APK file on your device
 - An ABI other than ARMv7 such as x86 or x86-64 (or a custom AAPT binary that supports ARMv7)
+- Patches such as [Revanced Patches](https://github.com/ReVanced/revanced-patches/releases)
 
-## ⏭️ Whats next
+## 🔹 Optional
```
Comment (oSumAtrIX) [inline docs/0_prerequisites.md:12]: `## 🔹 Optional` is at the same level as `## 🤝 Requirements` but should be semantically under `## 🤝 Requirements` as an "optional" requirement.

Regardless, integrations are not required to use ReVanced CLI at any stage, including patching. The usage page mentions that some patches may require integrations and how you can supply them.

**Comment by oSumAtrIX** [comment]
Thanks anyways.

**Comment by taku-nm** [comment]
Proposal: 
Take out integrations.
Include the hint to download patches, as they are required for the core function of the CLI and arguably less important than ADB.

**Comment by oSumAtrIX** [comment]
Both integrations and patches are not prerequisites for ReVanced CLI and, as such, don't belong on this page. Downloading anything is not required; for example, someone may also be building the patches themselves. The requirement to supply patches is documented on the usage page already.

**Pull request closed without merging**